R/as_tibble.R

```diff
@@ -230,7 +245,7 @@ as_tibble.matrix <- function(x, ..., validate = NULL, .name_repair = NULL) {
     validate <- NULL
   }
   colnames(m) <- names
-  as_tibble(m, ..., validate = validate, .name_repair = .name_repair)
+  as_tibble.data.frame(m, ..., validate = validate, .name_repair = .name_repair)
```

Comment:
This may fail if `as_tibble.data.frame()` chooses to call `NextMethod()` at some point (just hypothetically). What's the performance difference?

Reply:
No problem. I hadn't thought about the `NextMethod()` possibility. I will remove and clean up the stale comments. How do I proceed with the PR if the issue is closed? Create a new one?

Reply:
I think I understand. You closed the PR #572, but the original issue #567 is still open. I will just repeat the previous steps.